stage0/run.go

```diff
@@ -98,11 +99,12 @@ type RunConfig struct {
 
 // configuration shared by both Run and Prepare
```

Comment:
Didn't want to touch other things to make the patch showing only the related changes. If you prefer I'll fix it here.